src/matrixtools.jl

```diff
@@ -45,42 +55,62 @@ end
 function get_get_coo_to_com(mtype)
     if mtype == :csc
         get_coo_to_com = get_coo_to_csc
-    # elseif mtype == :cucsc
-    #     get_coo_to_com = get_coo_to_cucsc
     elseif mtype == :dense
         get_coo_to_com = get_coo_to_dense
-    # elseif mtype == :cudense
-    #     get_coo_to_com = get_coo_to_cudense
     end
 end
 
+function SparseMatrixCSC{Tv, Ti}(coo::SparseMatrixCOO{Tv,Ti}) where {Tv,Ti <: Integer}
```

Comment:
This overload `SparseMatrixCSC`'s constructor to build a CSC directly from a COO